packages/crayons-core/src/components/button/button.scss

```diff
@@ -1,50 +1,58 @@
 /**
   @prop --fw-button-min-width: minimum width for the button
   @prop --fw-button-label-vertical-padding: vertical padding for the button label
+  @prop --fw-button-height: The height of the normal button.
+  @prop --fw-button-small-height: The height of the small button.
+  @prop --fw-button-border-radius: The height of the small button.
+  @prop --fw-button-font-weight: The font weight of the button.
+  @prop --fw-button-font-size: The font size of the normal button.
+  @prop --fw-button-small-font-size: The font size of the small button.
```

Comment:
should we also provide customisation for other sizes of button like 'icon' , `icon-small` also?

Reply:
The icons are passed as slots so we can change the icon size from there.